script/1_0_2current_en_script/Arcueid/Day 6/06_00_ARC06_1_1.txt

```diff
@@ -3,58 +3,58 @@
 <Page4>
-"Good morning,# Nii-san.# Strangely enough, it feels like it has been a long time."# //Trailing space required
-No, well... isn't my sister a little too overpowering?
+"Good morning,# Nii-san.#It's strange, but I feel as though it's been quite some time since I last saw you."# //Trailing space required at end so the glue parses, but there is a forced line break after Nii-san so no space there. This actally gets injected badly by the current version of deepLuna due to how glue is implemented. Will be fixed. 不思議と、ずいぶん久しぶりな気がしますね
+No, well... isn't my little sister a bit too overpowering?
 <Page5>
 "Y-Yo. Morning, Akiha."
 "There is no need for greetings, so please take a seat over there.
 Yes, it seems that we have a lot to catch up on, don't you think?"
 <Page6>
 "............"
 ...Is it just my imagination, or does it look like she's having fun?
-It must be that. It's like a snake in front of a frog or something similar.
-Perhaps it would have been a good idea to just say hello and then leave.
+She must be. I'm like a deer caught in the headlights.//Not entirely sure what アレか is referring to. アレか。カエルを前にした蛇とか、そういうヤツか。
+Perhaps it would be a good idea to just respond with an appropriate greeting and then retreat.//やっぱりこのまま適当に挨拶をして撤退するのが良策なのか。
 <Page7>
 "―――Nii-san?"
-"Y, yeah, I'll take a seat!"
+"Y-Yeah, I'll take a seat right away!"
 <Page8>
-My naive idea of running away vanished with a single glance from Akiha.
+A single glare from Akiha was enough to dash my naive thoughts of escape.//逃げよう、なんて甘い考えは秋葉のひと睨みで霧散した。
 I sit down on the sofa slowly, holding back the pain in my left leg.
 <Page9>
 "I'm seated now."
 "You did not need to say that... Please do not dampen my spirit with your bizzare attitude. It's unfair."
 <Page10>
 "Why would it be unfair?"
-"Of course it's unfair. If you give me a face like that, I'll be rela――――#Ahem."
+"Of course it's unfair. If you make that face at me, I'll let my guard do――――#Ahem."//卑怯でしょう。そんな顔をされたら気が緩―――コホン
 <Page11>
-After stopping her statement with a cough, Akiha glares at me with an even sharper gaze.
+After truncating her statement with a cough, Akiha glares at me with an even sharper gaze than before.
 <Page12>
-"Nii-san. As prompt as this may be, would you mind telling me about what happened yesterday and the day before?"
+"Nii-san. I know this is sudden, but would you mind elucidating for me precisely what you have been up to both yesterday and the day before?"//兄さん。早速ですが、一昨日と昨日のお話を聞かせていただけませんか
 <Page13>
-Akiha picks up the teacup as she speaks with grace.
-Although her words resembled a question, it was most definitely closer to an order.
-But as her older brother, I can't let her hear such a story. I would much rather have Akiha hate me than worry about me.
+As she eloquently makes her remarks, Akiha picks up her teacup.//優雅に言いつつ、ティーカップを手に取る秋葉。
+Though her sentence may have ended with a question, there's no mistaking the fact that it is actually an order.//聞かせていただけませんか、などと言っているものの、間違いなく命令だ。
```

Comment:
sentence was "phrased as a question" since the question was actually at the start